tekton/rpmbuild/tekton.spec

```diff
@@ -606,6 +662,9 @@ cp -a cli-%{version}/docs/man/man1/* %{buildroot}%{_mandir}/man1
 %{_mandir}/*
 
 %changelog
+* Tue Sep 26 2019 Khurram Baig <kbaig@redhat.com> 0.4.0
```

Comment:
weird i am getting a 
`warning: bogus date in %changelog: Tue Sep 26 2019 Khurram Baig <kbaig@redhat.com> 0.4.0`

when building the spec

but i am not sure what exeactly

Reply:
Got it. Day is wrong. Haha.

Reply:
try using emacs's `rpm-add-changelog-entry` to add them and it should work (rpmlint should tell you the error too) https://github.com/stigbjorlykke/rpm-spec-mode/blob/master/rpm-spec-mode.el

Reply:
Installed that package. Thanks. It works/ :)